src/main/webapp/script.js

```diff
@@ -42,18 +42,6 @@ async function checkLog() {
   return loggedIn;
 }
 
-// Show a marker's settings area.
-function showSettings(contentId) {
```

Comment:
You sure this section should be deleted?

Reply:
O.... it still working.... magic? oO

Reply:
Yeah, I updated it and added it in the `mapScript.js`